frontend/src/container/NewDashboard/ComponentsSlider/index.tsx

```diff
@@ -1,146 +1,59 @@
-					version: data?.version || 'v3',
-					layout: [
+		handleToggleDashboardSlider(false);
+		const queryParamsLog = {
+			graphType: name,
+			widgetId: id,
+			[QueryParams.compositeQuery]: JSON.stringify({
+				...PANEL_TYPES_INITIAL_QUERY[name],
+				builder: {
+					...PANEL_TYPES_INITIAL_QUERY[name].builder,
+					queryData: [
 						{
-							i: id,
-							w: 6,
-							x: 0,
-							h: 3,
-							y: 0,
-						},
-						...(layouts.filter((layout) => layout.i !== PANEL_TYPES.EMPTY_WIDGET) ||
-							[]),
-					],
-					widgets: [
-						...(data?.widgets || []),
-						{
-							id,
-							title: '',
-							description: '',
-							isStacked: false,
-							nullZeroValues: '',
-							opacity: '',
-							panelTypes: name,
-							query:
-								name === PANEL_TYPES.LIST
-									? listViewInitialLogQuery
-									: PANEL_TYPES_INITIAL_QUERY[name],
-							timePreferance: 'GLOBAL_TIME',
-							softMax: null,
-							softMin: null,
-							selectedLogFields: [
-								{
-									dataType: 'string',
-									type: '',
-									name: 'body',
-								},
-								{
-									dataType: 'string',
-									type: '',
-									name: 'timestamp',
-								},
-							],
-							selectedTracesFields: [
-								...listViewInitialTraceQuery.builder.queryData[0].selectColumns,
-							],
+							...PANEL_TYPES_INITIAL_QUERY[name].builder.queryData[0],
+							aggregateOperator: LogsAggregatorOperator.NOOP,
+							orderBy: [{ columnName: 'timestamp', order: 'desc' }],
+							offset: 0,
+							pageSize: 100,
 						},
 					],
 				},
-			},
-			{
-				onSuccess: (data) => {
-					if (data.payload) {
-						handleToggleDashboardSlider(false);
-						const queryParamsLog = {
-							graphType: name,
-							widgetId: id,
-							[QueryParams.compositeQuery]: JSON.stringify({
-								...PANEL_TYPES_INITIAL_QUERY[name],
-								builder: {
-									...PANEL_TYPES_INITIAL_QUERY[name].builder,
-									queryData: [
-										{
-											...PANEL_TYPES_INITIAL_QUERY[name].builder.queryData[0],
-											aggregateOperator: LogsAggregatorOperator.NOOP,
-											orderBy: [{ columnName: 'timestamp', order: 'desc' }],
-											offset: 0,
-											pageSize: 100,
-										},
-									],
-								},
-							}),
-						};
+			}),
+		};
 
-						const queryParams = {
-							graphType: name,
-							widgetId: id,
-							[QueryParams.compositeQuery]: JSON.stringify(
-								PANEL_TYPES_INITIAL_QUERY[name],
-							),
-						};
+		const queryParams = {
+			graphType: name,
+			widgetId: id,
+			[QueryParams.compositeQuery]: JSON.stringify(
+				PANEL_TYPES_INITIAL_QUERY[name],
+			),
+		};
 
-						if (name === PANEL_TYPES.LIST) {
-							history.push(
-								`${history.location.pathname}/new?${createQueryParams(queryParamsLog)}`,
-							);
-						} else {
-							history.push(
-								`${history.location.pathname}/new?${createQueryParams(queryParams)}`,
-							);
-						}
-					}
-				},
-				onError: () => {
```

Comment:
do we need to handle onError condition?

Reply:
we are not making any APi calls this is just navigating to a new page. `OnError` condition comes in when the API fails or we have some operation that is suseptable to break.